tests/clients/test_fenda_client.py

```diff
@@ -0,0 +1,57 @@
+import copy
+
+import pytest
+import torch
+from flwr.common import Config
+
+from fl4health.clients.fenda_client import FendaClient
+from fl4health.model_bases.fenda_base import FendaModel
+from tests.clients.fixtures import get_fenda_client  # noqa
+from tests.test_utils.models_for_test import FeatureCnn, FendaHeadCnn
+
+
+@pytest.mark.parametrize("local_module,global_module,head_module", [(FeatureCnn(), FeatureCnn(), FendaHeadCnn())])
+def test_getting_parameters(get_fenda_client: FendaClient) -> None:  # noqa
+    torch.manual_seed(42)
+    fenda_client = get_fenda_client
+    config: Config = {}
+
+    assert isinstance(fenda_client.model, FendaModel)
+    params_local = [
+        copy.deepcopy(val.cpu().numpy()) for _, val in fenda_client.model.local_module.state_dict().items()
+    ]
+    params_global = [
+        copy.deepcopy(val.cpu().numpy()) for _, val in fenda_client.model.global_module.state_dict().items()
+    ]
+    fenda_client.get_parameters(config)
+
+    assert isinstance(fenda_client.old_local_module, torch.nn.Module)
+    assert isinstance(fenda_client.old_global_module, torch.nn.Module)
```

Comment:
One other check that might be useful here is to assert that `requires_grad` is false for each of the parameter components of these modules, just to be sure no gradient is being calculated for them.